backend/controllers/prisonerProfile/prisonerProfileBackLinkRedirect.ts

```diff
@@ -1,21 +1,13 @@
 import config from '../../config'
-import { isRedirectCaseLoad } from '../../utils'
```

Comment:
Removing `isRedirectCaseLoad` since this was based purely on exemption case loads which this PR is removing.  This means that this always redirects to the Prisoner Profile, which is why `backLink.ts` is removed above